review_practice/react-app/src/service/TokenManager.js

```diff
@@ -0,0 +1,12 @@
+const USER_INFO = 'userInfo';
+
+export const getToken = () =>
+    localStorage.getItem(USER_INFO) ? JSON.parse(localStorage.getItem(USER_INFO)).token : null;
```

Comment:
```suggestion
    localStorage.getItem(USER_INFO) ? JSON.parse(localStorage.getItem(USER_INFO))?.token : null;
```

로컬스토리지에 있는 값을 사용할 때는 항상 조심해주세요. 이상한 값으로 변조되면 이를 디버깅하기 힘들어지니 위와 같이 옵셔널 체이닝을 사용하여 안전하게 null을 리턴할 수 있도록합니다.